File: Selnium script for testing/Script with Python/test_scenario_44.py
from selenium import webdriver
from selenium.webdriver.common.by import By
import time

def execute_test_44():
    """
    Dummy test run for scenario 44
    """
    driver = webdriver.Chrome()
    driver.implicitly_wait(10)
    driver.get("https://dummy.example.com/page_44")

    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_990")
    time.sleep(2)
    driver.find_element(By.NAME, "password_field").send_keys("test_data_376")
    driver.find_element(By.CSS_SELECTOR, ".promo-code").send_keys("test_data_385")
    driver.find_element(By.PARTIAL_LINK_TEXT, "Profile").click()
    driver.find_element(By.ID, "checkout_btn").click()
    time.sleep(2)
    time.sleep(2)
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    driver.find_element(By.ID, "checkout_btn").click()

    driver.quit()

if __name__ == "__main__":
    execute_test_44()

File: Selnium script for testing/Script with Java/src/test/java/testscenario/TestScenario44.java
package testscenario;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import java.time.Duration;

public class TestScenario44 {
    public static void executeTest44() {
        /*
         * Dummy test run for scenario 44
         */
        WebDriver driver = new ChromeDriver();
        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
        driver.get("https://dummy.example.com/page_44");

        try {
            driver.findElement(By.linkText("Log Out")).sendKeys("test_data_173");
            driver.findElement(By.id("user_name")).sendKeys("test_data_731");
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            if (!driver.findElement(By.cssSelector(".promo-code")).isDisplayed()) {
                System.out.println("Element not displayed");
            }
            Thread.sleep(2000);
            driver.findElement(By.id("user_name")).sendKeys("test_data_403");
            driver.findElement(By.className("nav-item")).sendKeys("test_data_752");
            if (!driver.getTitle().contains("Dashboard")) {
                System.out.println("Dashboard not in title");
            }
            driver.findElement(By.id("user_name")).sendKeys("test_data_808");
        } catch (Exception e) {
            e.printStackTrace();
        } finally {
            driver.quit();
        }
    }
    
    public static void main(String[] args) {
        executeTest44();
    }
}
